### ISSUE TITLE
info command fails with no package manager plugin

### ISSUE THREAD

**Issue by bnigh** [intuit/auto#1014]
**Describe the bug**

When executing 

```bash
auto info
```

for a repository with no package manager plugin configured, the command fails with:

```bash
TypeError: Cannot read property 'startsWith' of undefined
    at Auto.constructor.prefixRelease (/snapshot/auto/node_modules/@auto-it/core/dist/auto.js:131:108)
    at Auto.info (/snapshot/auto/node_modules/@auto-it/core/dist/auto.js:325:55)
```

- Note: this causes other commands to fail when configured with the verbose (`-v`) flag, since `info` is executed for all commands conditional on the verbose flag

**To Reproduce**

- configure a repo with auto and ensure that no package manager plugins are used (even default npm plugin)
- execute 
  ```bash
  auto info
  ```

**Expected behavior**

This is failing because there is no hook configured for `getPreviousVersion` when trying to get the previous version. The commands should execute and give information for the rest of the fields. The `Current Version` value can be set as `not successful` and a msg can be given to suggest to the user to configure a package manager plugin.

**Screenshots**

n/a

**Environment information:**

```txt
"auto" version: v9.15.3
"git"  version: v2.15.2 (Apple Git-101.1)
"node" version: v10.17.0GHE version:    v2.19.3
```

**Additional context**

n/a

**Comment by bnigh** [comment]
I'm a bit busy today, but can likely take a look at this tomorrow

**Comment by adierkens** [comment]
:rocket: Issue was released in `v9.15.6` :rocket:
